client/src/modules/Globals/Styles/NavBar.css

```diff
@@ -10,7 +10,7 @@
 
   .navbar-searchbar-container {
     margin-left: 21px;
-    margin-right: 70px;
+    margin-right: 150px;
```

Comment:
In the future, we should try to stay away from hard-coding margins - not very responsive compared to relative widths.

Reply:
BUTTTT like probably easiest approach & solution for this case